doc/man/scc-hypervisor-collector.1.md

```diff
@@ -0,0 +1,203 @@
+---
+title: SCC-HYPERVISOR-COLLECTOR
+section: 1
+header: SCC Hypervisor Collector
+date: June 2022
+---
+# NAME
+**scc-hypervisor-collector** - Collect & Upload Hypervisor details
+to SUSE Customer Center
+
+# SYNOPSIS
+
+**scc-hypervisor-collector [<optional>...]**
+
+# DESCRIPTION
+
+The **scc-hypervisor-collector** collects details that are relevant
+to SUSE Customer Center (SCC) subscription compliance tracking, from
+the specified hypervisors.
+
+The collected details include the total system RAM, CPU architecture
```

Comment:
rendering shows 2 spaces after "architecture", I suspect you only want 1

Reply:
So that is `nroff -man` which by default tries to do centered text and will insert arbitrary spaces into lines as padding... The true man page rendering pipeline has a few more stages that improves things somewhat, and will also detect the true terminal width, which if wider than the default 80 chars, will avoid some of the line wrapping/hyphenation/padding issues you see on a narrower display.

Reply:
Fixed